bcda/main.go

```diff
@@ -55,20 +70,32 @@ func bulkRequest(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	err = validateBulkRequestHeaders(r.Header)
```

Comment:
Seems like a good candidate for a middleware to me. What do you think?

Reply:
Agree, that seems like a better place for it. I need to get a better sense of the typical Go application structure.